idaes/generic_models/unit_models/tests/test_surrogate_model.py

```diff
@@ -0,0 +1,118 @@
+##############################################################################
+# Institute for the Design of Advanced Energy Systems Process Systems
+# Engineering Framework (IDAES PSE Framework) Copyright (c) 2018-2020, by the
+# software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia
+# University Research Corporation, et al. All rights reserved.
+#
+# Please see the files COPYRIGHT.txt and LICENSE.txt for full copyright and
+# license information, respectively. Both files are also available online
+# at the URL "https://github.com/IDAES/idaes-pse".
+##############################################################################
+"""
+Generic template for a surrogate unit model.
+"""
+
+import pytest
+from pyomo.environ import ConcreteModel, Constraint, Var, value, SolverFactory
+from idaes.core import FlowsheetBlock
+from idaes.generic_models.unit_models import SurrogateModel
+from idaes.core.util.model_statistics import degrees_of_freedom
+from idaes.core.util.testing import get_default_solver
+
+__author__ = "Jaffer Ghouse"
+
+
+# -----------------------------------------------------------------------------
+# Get default solver for testing
+solver = get_default_solver()
+# -----------------------------------------------------------------------------
+
+m = ConcreteModel()
```

Comment:
It would be a good idea to wrap this in a fixture so that it isn't run on import.